fe/fe-core/src/main/java/com/starrocks/credential/CloudConfigurationConstants.java

```diff
@@ -84,4 +84,13 @@ public class CloudConfigurationConstants {
     public static final String GCP_GCS_SERVICE_ACCOUNT_PRIVATE_KEY_ID = "gcp.gcs.service_account_private_key_id";
     public static final String GCP_GCS_SERVICE_ACCOUNT_IMPERSONATION_SERVICE_ACCOUNT =
             "gcp.gcs.impersonation_service_account";
+
+    // Credential for HDFS
+    // TODO: Refactor the name of parameters
+    public static final String HDFS_AUTHENTICATION = "hadoop.security.authentication";
+    public static final String HDFS_USER_NAME = "username";
```

Comment:
Maybe we can consider using new parameters because the volume is a new thing, we don't need to be compatible with old parameters.

Reply:
> Maybe we can consider using new parameters because the volume is a new thing, we don't need to be compatible with old parameters.

We can keep it temporarily. Once the document is ready, we can refactor these parameters.

Reply:
got